activation_service_poc/start.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+cd ..
+make dockerbuild-go
+cd activation_service_poc
+
+IMAGE=$(docker images | head -n 2 | tail -n 1 | awk '{print $3}')
+sed -i "s/image.*/image:\ $IMAGE/g" docker-compose.yml
+TIME=$(date -u -d '2 minutes' "+%Y-%m-%dT%H:%M:%S%:z")
+sed -i "s/\"genesis-time\".*/\"genesis-time\"\:\"$TIME\"/g" config.standalone.client.json
+sed -i "s/\"genesis-time\".*/\"genesis-time\"\:\"$TIME\"/g" config.standalone.node-service.json
```

Comment:
Again this is probably not worth fixing for a PoC, but in general it's probably better to use `jq` tool to patch json files